Resources/views/Collector/db.html.twig

```diff
@@ -263,12 +265,16 @@
                                 </div>
 
                                 <div id="formatted-query-{{ i }}-{{ loop.parent.loop.index }}" class="sql-runnable hidden">
-                                    {{ query.sql|doctrine_pretty_query }}
+                                    {% set formatted_sql = query.sql|doctrine_pretty_query %}
+                                    {{ formatted_sql|raw }}
+                                    <button class="btn btn-sm" data-clipboard-text="{{ formatted_sql|striptags|e('html_attr') }}">Copy</button>
```

Comment:
Nice idea to add this one too but we need to html_entity_decode because htmlentities is applied on the formatted_sql so "<" is "&lt;" for example.